rcl/test/rcl/test_info_by_topic.cpp

```diff
@@ -308,6 +313,7 @@ TEST_F(CLASSNAME(TestInfoByTopicFixture, RMW_IMPLEMENTATION),
   default_qos_profile.deadline = {11, 0};
   default_qos_profile.liveliness_lease_duration = {20, 0};
   default_qos_profile.liveliness = RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_TOPIC;
+  default_qos_profile.avoid_ros_namespace_conventions = false;
```

Comment:
Thanks for this! I also noticed this test was flaky, because of this not being explicitly initialized.

Reply:
It makes sense to do this change in a separated PR, or better, split this in two PR?
(this comment https://github.com/ros2/rcl/pull/571#discussion_r376030648 should also be part of that PR/commit).

Reply:
I've created PR #572 for enabling the tests for Cyclone